tests/scaffoldings/dotd/custom/src/addons.yaml

```diff
@@ -3,6 +3,10 @@ other-doodba/odoo/src/private:
   - dummy_addon
   - product
 
+odoo/addons:
+  - *
+  - !hr_payroll
```

Comment:
The `!` character has a special meaning for YAML (it's breaking Travis here). Can't you use a different one? Otherwise, enclose it in quotes.

Also, as commented above, please append a `*`, so we can actually test that globs work as expected.

Reply:
ups, yes, of course :sweat_smile: 

Let's do this one first and see the effect in travis